lib/ansible/modules/cloud/amazon/ses_rule.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/python
+
+# -*- coding: utf-8 -*-
+#
+# (c) 2017, Ben Tomasik <ben@tomasik.io>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible. If not, see <http://www.gnu.org/licenses/>.
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = """
+---
+module: ses_rule
+short_description: Manages SES inbound receipt rules
+description:
+    - Allows creation, deletion, and management of SES receipt rules
+version_added: 2.4
+author:
+  - "Ben Tomasik (@tomislacker)"
+requirements: [ "boto3","botocore" ]
+options:
+  name:
+    description:
+      - The name of the receipt rule
+    required: True
+  ruleset:
+    description:
+      - The name of the receipt rule set
+    required: True
+  state:
+    description:
+      - Whether to create or destroy the receipt rule
+    required: False
+    default: present
+    choices: ["absent", "present"]
+  after:
+    description:
+      - Insert new rule before another (This only applies during creation)
+    required: False
+  enabled:
+    description:
+      - Whether the rule should be enabled or not
+    required: False
+    default: True
+  tls_required:
+    description:
+      - Whether inbound emails should require TLS
+    required: False
+    default: False
+  recipients:
+    description:
+      - Recipient specification(s)
+    required: True
+  actions:
+    description:
+      - Rule actions
+    required: True
+  scan_enabled:
+    description:
+      - Whether inbound emails should get virus scanned
+    required: False
+    default: False
+extends_documentation_fragment: aws
+"""
+
+EXAMPLES = """
+
+# Pushes emails to S3 that are received for any address @mydomain.com as well
+# as any address @<any subdomain>.mydomain.com
+- name: Create catch-all rule
+  ses_rule:
+    name: main-rule
+    ruleset: default-rule-set
+    recipients:
+      - '.mydomain.com'
+      - 'mydomain.com'
+    actions:
+      - S3Action:
+          BucketName: my-bucket
+
+"""
+
+RETURN = ""
+
+try:
+    import boto3
+    HAS_BOTO3 = True
+except ImportError:
+    HAS_BOTO3 = False
+
+try:
+    import botocore
```

Comment:
Looks like you went a bit far addressing this - now you don't import botocore at all

You need the above code